### ISSUE TITLE
Crashing app when used the imagepicker in Asus Zenphone 

### ISSUE THREAD

**Issue by GhAc-2017** [esafirm/android-image-picker#328]
## Expected Behavior
Imagepicker should show gallery and help pick an image

## Actual Behavior
app crashing when calling the imagepicker to show gallery(permission allowed)
My code - > which seems to work in many devices, but stumbled upon this Zenphone model
```
ImagePicker.create(Profile.this) // Activity or Fragment
    .limit(1)
    .start();
```
Error - 
```
Fatal Exception: java.lang.NullPointerException
Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkNotNullParameter, parameter name
com.esafirm.imagepicker.model.Image.<init> (Unknown Source:2)
com.esafirm.imagepicker.features.fileloader.DefaultImageFileLoader$ImageLoadRunnable.run (DefaultImageFileLoader.java:159)
java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1167)
java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:641)
java.lang.Thread.run (Thread.java:764)
```
## Steps to Reproduce the Problem

  1. Just call the Imagepiker method as per docs
  2. it will try to open the gallery
  3. crash

## Specifications

  - Image Picker Version: 2.4.1
  - Android OS: 9
  - Phone: Asus ZenFone Max M2 (ZB633KL) (WW) / Max M2 (ZB632KL) (IN)

**Comment by simon-the-canadian** [comment]
I'm getting this same crash from a user on production.
Their devices is HUAWEI Y6 2019 and its also running Android 9. Not sure if it could be Android 9 related, or maybe they have a bad image on their device that's loading as `null`?
I'm also using version `2.4.1`

**Comment by esafirm** [comment]
This should be fixed on [2.4.5](https://github.com/esafirm/android-image-picker/releases/tag/2.4.5)
Also, if you can test the [3.0.0-alpha1](https://github.com/esafirm/android-image-picker/releases/tag/3.0.0-alpha1) version, it would help me so much. We're going full kotlin now 🔥
